File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++){
            // if (target - nums[i])
            for (int j = 1; j < nums.length && i != j; j++) {
                if (nums[i] + nums[j] == target){
                    int[] sol = {i, j};
                    Arrays.sort(sol);
                    return sol;
                }
            }
        } 
        return null;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        const starters = new Map();
        const sorted = [...new Set(nums.sort((a, b) => a - b))];
        console.log(sorted);
        let currentHighest;
        let total = 0;
        for (let i = 0; i < sorted.length; i++) {
            if (!sorted.includes(sorted[i] - 1) && !starters.get(sorted[i])) {
                starters.set(sorted[i], 1);
                currentHighest = sorted[i];
                if (total === 0) {
                    total = 1
                }
            } else {
                const counter = starters.get(currentHighest) + 1;
                starters.set(currentHighest, counter);
                if (total < counter) {
                    total = counter;
                }
            }
        }
        console.log(starters)
        return total;
    }
}
